tests/components/test_dialogue.py

```diff
@@ -291,8 +291,10 @@ def test_convo_params_are_cleared(kwik_e_mart_nlp, kwik_e_mart_app_path):
     [
         ("en", "en_GB", {"lang": "EN", "latent": True, "locale": "en_GB"}),
         ("es", "en_US", {"latent": True, "locale": "en_US"}),
-        (None, None, {"latent": True, "locale": "en_US", "lang": "EN"}),
+        (None, None, {"latent": True, "locale": "en_CA", "lang": "EN"}),
```

Comment:
why change?

Reply:
I have updated the kwik-e-mart test application to use the `en_CA` locale